components/Contact.vue

```diff
@@ -0,0 +1,167 @@
+              </p>
+              <article>
+                <label class="hidden" for="firstname">First Name</label>
+                <input
+                  id="name"
+                  v-model="firstname"
+                  type="text"
+                  :placeholder="contact.firstnamePlaceholder"
+                  name="firstname"
+                  size="100"
+                >
+              </article>
+              <article>
+                <label class="hidden" for="lastname">Last Name</label>
+                <input
+                  id="name"
+                  v-model="lastname"
+                  type="text"
+                  :placeholder="contact.lastnamePlaceholder"
+                  name="lastname"
+                  size="100"
+                >
+              </article>
+              <article>
+                <label class="hidden" for="email">Email</label>
+                <input
+                  id="email"
+                  v-model="email"
+                  type="email"
+                  :placeholder="contact.emailPlaceholder"
+                  name="email"
+                  size="30"
+                  required
+                >
+              </article>
+              <article>
+                <label for="msg" class="hidden">Message</label>
+                <textarea
+                  id="msg"
+                  v-model="message"
+                  :placeholder="contact.messagePlaceholder"
+                  name="message"
+                  cols="40"
+                  rows="3"
+                />
+                <div class="btn-wrap  text-center">
+                  <button id="submit" class="btn btn-odin btn-odin-color" name="submit" type="submit">
+                    {{ contact.buttonText }}
+                  </button>
+                </div>
+              </article>
+            </form>
+          </div>
+        </article>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import contact from '../static/content/contact.json'
+
+export default {
+  name: 'Contact',
+  data() {
+    return {
+      contact: contact,
+      firstname: '',
+      lastname: '',
+      email: '',
+      message: ''
+    }
+  },
+  methods: {
+    sendEmail: function () {
+      const emailData = {
+        email: this.email,
+        firstname: this.firstname,
+        lastname: this.lastname,
+        message: this.message
+      }
+      this.$store.dispatch('contactUs', emailData)
+      this.firstname = ''
+      this.lastname = ''
+      this.email = ''
+      this.message = ''
+    },
+    createTicket: function () {
+      const ticketInfo = {
+        email: this.email,
+        firstname: this.firstname,
+        lastname: this.lastname,
+        message: this.message
+      }
+      this.$store.dispatch('contactTicket', ticketInfo)
+    },
+    async onSubmit() {
+      try {
+        // eslint-disable-next-line
+        const token = await this.$recaptcha.execute('login')
```

Comment:
Please confirm : I think doing this may "hang" the UI when user wait for login. If that's the case, best way is use spinner and may add a cancel button in case login timeout.